test/Kestrel.Core.Tests/Http2ConnectionTests.cs

```diff
@@ -1065,12 +1065,6 @@ public async Task DATA_Received_StreamClosedImplicitly_ConnectionError()
         [Fact]
         public async Task DATA_Received_NoStreamWindowSpace_ConnectionError()
         {
-            // I hate doing this, but it avoids exceptions from MemoryPool.Dipose() in debug mode. The problem is since
```

Comment:
There are more of these in Http2StreamTests https://github.com/aspnet/KestrelHttpServer/blob/bd3a8a3511238a87c61d9ed798db373c1b88dd78/test/Kestrel.Core.Tests/Http2StreamTests.cs#L967